--- cassandra-env.sh-orig	2013-11-01 22:39:23.899752001 +0000
+++ cassandra-env.sh	2013-11-01 22:39:58.067752000 +0000
@@ -72,8 +72,8 @@
 # times. If in doubt, and if you do not particularly want to tweak, go with
 # 100 MB per physical CPU core.
 
-#MAX_HEAP_SIZE="4G"
-#HEAP_NEWSIZE="800M"
+MAX_HEAP_SIZE="4G"
+HEAP_NEWSIZE="800M"
 
 if [ "x$MAX_HEAP_SIZE" = "x" ] && [ "x$HEAP_NEWSIZE" = "x" ]; then
     calculate_heap_sizes
